pom.xml

```diff
@@ -27,14 +27,14 @@
     <project.checkout>scm:git:git@github.com:querydsl/querydsl.git</project.checkout>
     
     <!-- deps -->
-    <derby.version>10.10.1.1</derby.version>
-    <hsqldb.version>2.3.1</hsqldb.version>
+    <derby.version>10.11.1.1</derby.version>
+    <hsqldb.version>2.3.2</hsqldb.version>
     <h2.version>1.4.178</h2.version>
```

Comment:
1.4.186 is the latest version for H2.

There are some other dependencies that could be updated. Separate PR, perhaps?